.github/workflows/build_shadow.yml

```diff
@@ -22,24 +19,24 @@ jobs:
       options: '--shm-size="1g"'
     strategy:
       matrix:
-        container: ['ubuntu:18.04', 'centos:7'] #, 'ubuntu:16.04', 'ubuntu:20.04', 'debian:10-slim', 'fedora:32', 'centos:8']
-        cc: ['gcc'] #, 'clang']
+        container: ['ubuntu:18.04', 'ubuntu:20.04', 'debian:10-slim', 'fedora:32', 'centos:7', 'centos:8'] # 'ubuntu:16.04'
```

Comment:
Could you add a comment with a link to the issue for ubuntu 16.04?

Reply:
Added